project/ScalazBuild.scala

```diff
@@ -21,7 +21,6 @@ object Scalaz {
     "-Yno-adapted-args",
     "-Xlint:_,-type-parameter-shadow",
     "-Xsource:2.13",
-    "-Ywarn-dead-code",
```

Comment:
that's a pretty terrible removal. Why is it necessary?

Reply:
#89